src/tests/cli_tests.py

```diff
@@ -0,0 +1,454 @@
+#!/usr/bin/python
```

Comment:
On macOS I have only `/usr/bin/python2.7` and `/usr/local/bin/python2`. So, maybe, when integrating this to Travis we should better run it as 'python2 cli_tests.py all'? Or there are other ideas how to make it to work everywhere.